src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -90,10 +90,33 @@
     owner: root
     group: root
   no_log: true
-  when: not fake_ssl_cert and nginx_key_value
+  when: not fake_ssl_cert and nginx_key_value | default(None)
```

Comment:
I think these changes isn't necessary—`nginx_key_value` and `nginx_combined_cert_value` are automatically added to .generated.yml if they're not included in `proxy.yml`.